plugins/ida/bap_ida_service.ml

```diff
@@ -79,8 +81,24 @@ let setup_headless_env path =
       if old_path <> "" then Unix.putenv var old_path
     with _ -> ()
 
+let cleanup_minidump () =
+  let is_dump x = String.equal (Filename.extension x) ".dmp" in
+  let dump_path = "/tmp/ida" in
+  if Sys.file_exists dump_path then
+    FileUtil.ls dump_path  |>
+    List.filter ~f:is_dump |> function
+    | [] -> ()
+    | files ->
+      info "ida minidump is not empty";
+      let lock = sprintf "%s/lock" dump_path in
+      let lock = Unix.openfile lock Unix.[O_RDWR; O_CREAT] 0o640 in
```

Comment:
since the lockfile is global it should be readable (and thus shareable) by all users, so provide corresponding rights.

Consider the scenario when BAP is run by two different users, the first user will create a lock file, while the second will not be able to graph the lock as she will fail on `open`.